env.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
```

Comment:
What is this file, and why do we need it?

Regardless, it must be in strict mode if it's not babel-transpiled; and ideally it should be babel-transpiled.